sharding/collator/collator.go

```diff
@@ -59,7 +59,7 @@ func checkSMCForCollator(c client.Client, head *types.Header) error {
 	period := big.NewInt(0).Div(head.Number, big.NewInt(sharding.PeriodLength))
 	for s := int64(0); s < sharding.ShardCount; s++ {
 		// Checks if we are an eligible collator according to the SMC
-		addr, err := c.SMCCaller().GetEligibleCollator(&bind.CallOpts{}, big.NewInt(s), period)
+		addr, err := c.SMCCaller().Get_eligible_collator(&bind.CallOpts{}, big.NewInt(s), period)
```

Comment:
How did this happen? Please change it back

Reply:
new method name from SMC (phase 1) bindings #78

Reply:
ok I'll bring it up with @enriquefynn